requirements_builder/cli.py

```diff
@@ -32,13 +32,24 @@
     '--req', '-r', default=None, help='Requirements file.',
     metavar='PATH', type=click.Path(
         exists=True, dir_okay=False, readable=True, resolve_path=True))
+@click.option(
+    '--output', '-o', default=None, help='Output file.',
+    type=click.File('w'))
```

Comment:
can you use `stdout` as default (`-`) - see http://click.pocoo.org/5/arguments/#file-args